README.md

```diff
@@ -1,4 +1,329 @@
+xpath | For elements lookup Xpath strategy the driver uses the same XML tree that is generated by page source API. Only Xpath 1.0 is supported (based on xmllib2). | `By.xpath("//XCUIElementTypePopUpButton[@value="Regular" and @label="type face"]")`
+
+Check the [integration tests](/test/functional/find-e2e-specs.js) for more examples on different location strategies usage.
+
+
+## Platform-Specific Extensions
+
+Beside of standard W3C APIs the driver provides the following custom command extensions to execute platform specific scenarios:
+
+### macos: click
+
+Perform click gesture on an element or by relative/absolute coordinates
+
+#### Arguments
+
+Name | Type | Required | Description | Example
+--- | --- | --- | --- | ---
+element | string | if `x` or `y` are unset | Unique identifier of the element to perform the click on. Either this property or/and x and y must be set. If both are set then x and y are considered as relative element coordinates. If only x and y are set then these are parsed as absolute coordinates. | 21045BC8-013C-43BD-9B1E-4C6DC7AB0744
+x | number | if `y` is set or `element` is unset | click X coordinate | 100
+y | number | if `y` is set or `element` is unset | click Y coordinate | 100
+keyModifierFlags | number | no | if set then the given key modifiers will be applied while click is performed. See the official documentation on [XCUIKeyModifierFlags enumeration](https://developer.apple.com/documentation/xctest/xcuikeymodifierflags) for more details | 1 << 1 `logical or` 1 << 2
+
+### macos: clickAndHold
+
+Perform long click gesture on an element or by relative/absolute coordinates
+
+#### Arguments
+
+Name | Type | Required | Description | Example
+--- | --- | --- | --- | ---
+element | string | if `x` or `y` are unset | Unique identifier of the element to perform the long click on. Either this property or/and x and y must be set. If both are set then x and y are considered as relative element coordinates. If only x and y are set then these are parsed as absolute coordinates. | 21045BC8-013C-43BD-9B1E-4C6DC7AB0744
+x | number | if `y` is set or `element` is unset | long click X coordinate | 100
+y | number | if `y` is set or `element` is unset | long click Y coordinate | 100
+duration | number | yes | The number of float seconds to hold the mouse button | 2.5
+keyModifierFlags | number | no | if set then the given key modifiers will be applied while long click is performed. See the official documentation on [XCUIKeyModifierFlags enumeration](https://developer.apple.com/documentation/xctest/xcuikeymodifierflags) for more details | 1 << 1 `logical or` 1 << 2
```

Comment:
`1 << 1` logical or `1 << 2`? (for example of keyModifierFlags. The positions of `)